torch/ao/quantization/quantizer/xnnpack_quantizer_utils.py

```diff
@@ -120,10 +120,14 @@ def get_input_act_qspec(quantization_config: Optional[QuantizationConfig]):
     if quantization_config.input_activation is None:
         return None
     quantization_spec: QuantizationSpec = quantization_config.input_activation
-    assert quantization_spec.qscheme in [
-        torch.per_tensor_affine,
-        torch.per_tensor_symmetric,
-    ]
+    assert (
+        quantization_spec.qscheme
+        in [
+            torch.per_tensor_affine,
+            torch.per_tensor_symmetric,
+        ]
+        or quantization_spec.is_dynamic
+    ), "Invalid qscheme for activation in quantization spec"
```

Comment:
are you using this util? this is supposed to be xnnpack only, so I think you should copy paste this util function

Reply:
We are using it. I found this change is not needed. So, I just reverted it.